modules/liquidity-pools/src/lib.rs

```diff
@@ -300,7 +316,21 @@ impl<T: Trait> Module<T> {
 	) -> DispatchResult {
 		ensure!(Self::is_owner(pool_id, who), Error::<T>::NoPermission);
 		let mut pool = Self::liquidity_pool_options(&pool_id, &currency_id).unwrap_or_default();
-		pool.additional_collateral_ratio = ratio;
+		let min_ratio =
```

Comment:
shouldn't enforce in setter but in getter instead
e.g.
min ratio is 30%
pool set ratio to 20%
the set should success but when reading the ratio it should return 30%
and then min ratio could change to 10%
now reading the ratio should be 20%